### ISSUE TITLE
Some arguments ignored on Windows

### ISSUE THREAD

**Issue by belg4mit** [jarun/imgp#56]
Hello,

I just tried running master (v2.9) on Windows and it mostly works, but some arguments seem to be ignored? -r is heeded and -x is processed but ignored while -d, -m and -w have no effect. However, the flags appears to be set properly internally, per the output from this basic [debug.patch](https://github.com/jarun/imgp/files/14562330/debug.patch):

    D:\Tools\>c:\Progra~1\Python\python -V
    Python 3.9.1
    D:\Tools\>c:\Progra~1\Python\python imgp-master\imgp --mute --quality 90 -x 800x800 --overwrite -r imgp
    90
    True
    True

Non-flag heeding invocation output pre-debug patch (identical with short or long-form parameters):

    D:\Tools\>c:\Progra~1\Python\python imgp-master\imgp --mute --quality 90 -x 800x800 --overwrite -r imgp.test
    imgp.test\10_PHOTO1\R_12fwK7duntjbSI9_IMG_0145.jpg
    1280x960 -> 1280x960
    396696 bytes -> 173783 bytes

Any ideas?

--JAPH

**Comment by superego1984** [comment]
@belg4mit, I don't think it works properly under windows. As you can see, you want to resize the image to 800x800, however the output always gives you the same resolution of than the original image. 

_D:\Tools\>c:\Progra~1\Python\python imgp-master\imgp --mute --quality 90 -x **800x800** --overwrite -r imgp.test
imgp.test\10_PHOTO1\R_12fwK7duntjbSI9_IMG_0145.jpg
1280x960 -> **1280x960**
396696 bytes -> 173783 bytes_

I have the same issue when I use it under my win PC, all the global variables cannot be sent to the sub methods, like **resize_image**

**Comment by jarun** [comment]
Sorry guys, I don't have Windows. Please raise a PR is you find a solution.
